apiv2/schema/schema.yaml

```diff
@@ -553,13 +553,17 @@ classes:
         multivalued: true
         inverse: AnnotationFile.alignment
         range: AnnotationFile
+        annotations:
+          cascade_delete: true
```

Comment:
I tried to take a middle-ground approach in choosing which objects to delete when the parents got deleted, but this could use a look from @manasaV3 / @uermel